content/search-github/github-code-search/using-github-code-search.md

```diff
@@ -40,11 +40,15 @@ For more information about the search syntax of code search, see "[AUTOTITLE](/s
 ## Creating and managing saved searches
 
 1. In the top navigation of {% data variables.product.prodname_dotcom_the_website %}, click the search bar and type `saved:`.
-1. Under the search bar, the "Saved searches" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Create saved search**.
+![GitHub top navigation, user typing 'saved:' in search bar.](/assets/images/help/search/navigatetosearch.png)
+1. Under the search bar, the "Saved queries" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
+!['Saved queries' section, user clicking 'Manage saved searches'.](/assets/images/help/search/selectmanagesavedsearch.png)
 1. In the pop-up window, fill out the name you want for your query and the query that you want to save. Click **Create saved search**.
```

Comment:
```suggestion
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
```

Small change to some pre-existing text to better align with the [suggested terms](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#terms-to-use-or-avoid) in our style guide, and to split these actions into separate steps. Nothing to do with your edits! 😁